test/src/Support/StringifyTest.php

```diff
@@ -26,10 +26,11 @@ public function testOptions()
             new Option('start-from', null, Option::VALUE_REQUIRED, 'Start execution from this task'),
             new Option('log', null, Option::VALUE_REQUIRED, 'Write log to a file'),
             new Option('profile', null, Option::VALUE_REQUIRED, 'Write profile to a file',),
+            new Option('ansi', null, Option::VALUE_OPTIONAL, 'Force ANSI output',),
         ]);
 
-        self::assertEquals("--option 'env=prod' --limit 1 -vvv", Stringify::options(
-            new ArgvInput(['deploy', '-o', 'env=prod', '-l1'], $definition),
+        self::assertEquals("--option 'env=prod' --option ansi --limit 1 -vvv", Stringify::options(
```

Comment:
I think second option is redundant.

Reply:
I tried following your feedback, but the [test output](https://github.com/deployphp/deployer/pull/2402/checks?check_run_id=1845210970) indicates otherwise.

Reply:
@antonmedv do you agree with my latest test case? Can we merge this merge request?

Reply:
Ping @antonmedv

Reply:
Sure. Sorry for delay.

Reply:
Actually unit tests are failing.

Reply:
This caused a rebase with master where the changelog was not sorted. Fixed.